Executer/CommandExecuter.php

```diff
@@ -60,43 +56,36 @@ public function execute($commandString)
         $result = $output->getBuffer() || ob_get_contents();
```

Comment:
The `$result` variable seems not be used. It's obvious not your fault but mine as it has already been this way before.

A few years ago there were some commands which did not use the Output interface but `echo`'ed directly to stdout. Thats why I wrapped the command execution in an output buffer. But apparently I missed passing on the collected output.

I think we should either remove the line or concatenate the result of `ob_get_contents()` to `$output->getBuffer()` instead of choosing one of them.